docs/release_notes/pending_release.rst

```diff
@@ -20,6 +20,11 @@ Utils
     duplicated across `PerturbImage` implementations. Added both batch and
     streaming versions of this utility.
 
+Documentation
```

Comment:
There is no hard constraint on order of things listed here but my OCD compels me to mention that alphabetically ordered sections is my preference.

Reply:
I have fixed this now.